fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -1412,7 +1419,10 @@ private void computeMultiCastFragmentParams() throws Exception {
                 multi.getDestFragmentList().get(i).setOutputPartition(params.fragment.getOutputPartition());
 
                 PlanNodeId exchId = sink.getExchNodeId();
+                PlanNode exchNode = PlanNodeUtils.findPlanNodeFromPlanNodeId(destFragment.getPlanRoot(), exchId);
                 Preconditions.checkState(!destParams.perExchNumSenders.containsKey(exchId.asInt()));
+                Preconditions.checkState(exchNode != null, "exchNode is null");
+                Preconditions.checkState(exchNode instanceof ExchangeNode, "exchNode is not ExchangeNode");
```

Comment:
print exchNode's id and type in check message

Reply:
done